package.json

```diff
@@ -52,48 +50,42 @@
     "moment": "^2.16.0",
     "moment-range": "^2.2.0",
     "moment-timezone": "^0.5.13",
-    "react-click-outside": "^2.3.0"
+    "prop-types": "^15.5.6",
+    "react": "^15.6.2",
+    "react-click-outside": "^2.3.0",
+    "react-dom": "^15.6.2"
   },
   "devDependencies": {
     "@salesforce-ux/design-system": "2.3.1",
-    "babel-cli": "^6.14.0",
-    "babel-core": "^6.14.0",
-    "babel-eslint": "^6.1.2",
-    "babel-jest": "^16.0.0",
-    "babel-loader": "^6.2.5",
-    "babel-plugin-transform-dev-warning": "^0.1.0",
-    "babel-preset-es2015": "^6.14.0",
-    "babel-preset-react": "^6.11.1",
-    "babel-preset-stage-0": "^6.5.0",
-    "copy-webpack-plugin": "^3.0.1",
-    "css-loader": "^0.24.0",
-    "enzyme": "^2.8.2",
-    "eslint": "^3.19.0",
-    "eslint-config-airbnb": "^14.1.0",
-    "eslint-import-resolver-webpack": "^0.6.0",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^4.0.0",
-    "eslint-plugin-react": "^6.10.3",
-    "file-loader": "^0.9.0",
-    "fs-extra": "^0.30.0",
-    "highlight.js": "^9.6.0",
-    "jest-cli": "^19.0.2",
-    "json-loader": "^0.5.4",
-    "marked": "^0.3.6",
-    "node-sass": "^3.8.0",
-    "prop-types": "^15.5.6",
-    "raw-loader": "^0.5.1",
-    "react": "^15.5.3",
-    "react-addons-test-utils": "^15.5.1",
-    "react-docgen": "^2.10.0",
-    "react-dom": "^15.5.3",
-    "react-hot-loader": "^1.3.0",
-    "react-router": "^3.0.5",
-    "react-test-renderer": "^15.5.4",
-    "recast": "^0.11.12",
-    "sass-loader": "^4.0.0",
-    "style-loader": "^0.13.1",
-    "webpack": "^1.13.2",
-    "webpack-dev-server": "^1.15.0"
+    "@storybook/addon-actions": "^3.2.6",
+    "@storybook/addon-info": "^3.2.11",
+    "@storybook/addon-knobs": "^3.2.10",
+    "@storybook/addon-links": "^3.2.6",
+    "@storybook/react": "^3.2.8",
+    "@storybook/storybook-deployer": "^2.0.0",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "^8.0.1",
+    "babel-jest": "^21.2.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-react-constant-elements": "^6.23.0",
```

Comment:
This introduces some bugs and warnings. For example in `InputRaw`, it will hoist the right `ButtonIcon` out of the `renderIconRight`. As a result, this throws a warning because the icon button  is created in any case, even if there is no icon prop.

Also, it throws a bunch of undefined element warnings: `Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined.`.
Please see https://github.com/babel/babel/issues/5577

Browsing the babel issue tracker for issues with these plugin yields a bunch of results. Might not be a good choice using this optimization.

Reply:
@extronics 👍 will remove